components/canvas/webgl_thread.rs

```diff
@@ -183,6 +212,13 @@ impl<VR: WebVRRenderHandler + 'static> WebGLThread<VR> {
         self.webvr_compositor.as_mut().unwrap().handle(command, texture.map(|t| (t.texture_id, t.size)));
     }
 
+    /// Handles a request to prevent a GL context from being modified until further notice.
+    fn handle_prevent_modification(&mut self, context_id: WebGLContextId) {
```

Comment:
Should this assert that the context is locked? Otherwise there's no guarantee that the context would be unlocked, right?